README.md

```diff
@@ -15,7 +15,7 @@ Linux based systems.
 
 ## Development
 
-The recommended way to set up Katello for development to use the [katello-devel-installer](https://github.com/Katello/katello-installer#development-usage) directly or via [katello-deploy](https://github.com/Katello/katello-deploy#development-deployment).
+The recommended way to set up Katello for development to use the [katello-devel-installer](https://github.com/Katello/katello-installer#development-usage) directly or via [katello-deploy](https://github.com/Katello/katello-deploy#development-deployment) if you prefer to use Vagrant. If you run into issues during the development setup, feel free to ask for help on #theforeman-dev or via the foreman-dev mailing list.
```

Comment:
Should I change the doc to only mention the two styles of running katello-deploy, and remove the reference to katello-devel-installer?